Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -69,5 +73,16 @@ def commit(self):
             distances = inv(distances)
         self.send("Distances", distances)
 
+    def send_report(self):
+        norm, normopt = self.normalization_method, self.normalization_options
+        inv, invopt = self.inversion_method, self.inversion_options
+        parts = []
+        if inv:
+            parts.append('inversion ({})'.format(invopt[inv][0]))
+        if norm:
+            parts.append('normalization ({})'.format(normopt[norm][0]))
+        self.report_items('Model parameters',
+                          (('Transformation', ', '.join(parts).capitalize() or 'None'),))
```

Comment:
I don't mind myself, but with 89 characters, I think this line is now 10 printable characters too long for @janezd's taste.